index.html

```diff
@@ -92,7 +130,40 @@ <h3>We put recipes to the <br>test, so you can<br>savor the success!</h3>
     <div class="img2"></div>
   </div>
 </body>
+<!-- Reviews Section -->
+<div class="reviews" id="reviews">
+  <h1>Reviews</h1>
+  <div id="review-list"></div>
+</div>
+
+<script>
+  const reviews = [
+    { username: "Username1", date: "2024-12-12 13:30", stars: 5, review: "Amazing recipes! I tried the Burnt Rice Bowl and it was
+    { username: "Username2", date: "2024-11-24 14:45", stars: 5, review: "Loved it! Perfect and easy for beginners. " },
+    { username: "Username3", date: "2023-09-22 20:15", stars: 4, review: "Great unique recipes but a bit hard to follow" }
+  ];

```

Comment:
This data needs to be read from the recipes.json file for each individual recipe